CMakeLists.txt

```diff
@@ -357,6 +358,7 @@ target_link_libraries(codegen_internal PUBLIC
   ${LIBCUPTI}
   ${TORCH_LIBRARIES}
   dl
+  cuda
```

Comment:
needed for `cuMemGetAddressRange`, otherwise getting an "invalid context" error. But according to https://github.com/NVIDIA/Fuser/issues/3907 there is clearly an issue with how we load driver API in general, so I'm linking to it directly in the meantime